praw/__init__.py

```diff
@@ -1750,19 +1750,27 @@ def set_flair_csv(self, subreddit, flair_mapping):
             raise errors.ClientException('flair_mapping must be set')
         item_order = ['user', 'flair_text', 'flair_css_class']
         lines = []
+        comma_response = []
         for mapping in flair_mapping:
             if 'user' not in mapping:
                 raise errors.ClientException('flair_mapping must '
                                              'contain `user` key')
-            lines.append(','.join([mapping.get(x, '') for x in item_order]))
+            if any((mapping.get(x, '').find(',') != -1) for x in mapping):
+                comma_response.append(self.set_flair(subreddit, mapping.get('user'), mapping.get('flair_text'),
```

Comment:
I don't like that this has a hidden one request per one flair possibility. I would rather have the user explicitly say via an argument that they're okay with such requests, otherwise, we can maybe create a new exception that contains a list of flair not set, along with the response of setting the flair that could be set.